src/gradleinttest.sh

```diff
@@ -32,6 +32,10 @@ gradleinttest () {
   if [[ "${test_to_run}" == "testConnected" ]]; then
     google_data_project="broad-jade-integration-data"
     echo "Running ${test_to_run} test with data project: ${google_data_project}"
+  else if [[ "${test_to_run}" == "testPerf" ]]; then
+    google_project="broad-jade-perf"
+    google_data_project="broad-jade-perf-data"
+    echo "Running ${test_to_run} test with project ${google_project} and data project: ${google_data_project}"
   else
     google_data_project=""
```

Comment:
not part of your pr, but why is this an empty string?

Reply:
I think it's unset so that it can be overwritten by data project selected for the integration tests (e.g. int-3-data or int-4-data)